### ISSUE TITLE
Build baseline perf app(s) and measure

### ISSUE THREAD

**Issue by kmelmon** [microsoft/react-native-windows#4872]
As part of an effort to establish a baseline of perf numbers for react-native-windows, we should build simplified scenario apps and get essential perf data from them.

Part of this effort should include identifying which are the most basic, essential scenarios we want to measure, and then build some number of apps to exercise those scenarios.

We have done this in the past for XAML (eg GridViewPan and several other simple apps ran as part of CPGs), so we can borrow from those for ideas.

We should also focus on bundle loading and javascript performance as those are key metrics that won't be captured for free by measuring XAML performance.

**Comment by chrisglein** [comment]
May be overlap with #4874, but that's not a min app, and we probably want that.
Assigning to @kmelmon to flesh out the thinking here and explore what kinds of apps we want.

**Comment by jonthysell** [comment]
As we're working on perf stuff now, should this be moved into 0.67 @chrisglein? Do we want to still track this specific case (generic base apps) separate from the work we _know_ we'll be doing with https://github.com/microsoft/react-native-windows-samples/issues/513

**Comment by chrisglein** [comment]
> As we're working on perf stuff now, should this be moved into 0.67 @chrisglein Chris Glein FTE? Do we want to still track this specific case (generic base apps) separate from the work we _know_ we'll be doing with [microsoft/react-native-windows-samples#513](https://github.com/microsoft/react-native-windows-samples/issues/513)

Yep, moved to 0.67. After initial investigation we'll have a better idea of what baseline/sample apps we need to build (vs. using existing).

**Comment by jonthysell** [comment]
@marlenecota Do you think this will make 0.69 or should it be moved to 0.70?
